book_maker/cli.py

```diff
@@ -280,6 +286,7 @@ def main():
         options.resume,
         language=language,
         model_api_base=model_api_base,
+        deployment_id=options.deployment_id,
```

Comment:
let us put it in here since other translator do not has this attr
https://github.com/yihong0618/bilingual_book_maker/blob/fa9e375a8ec8c94759dfad0c1c7f7704b1ab99ec/book_maker/cli.py#L295-L307

Reply:
I have tried to do this, but later, I didn't know how to receive and access the `deployment_id` in `EPUBBookLoader` or `TXTBookLoader`. I have tried for a long time without success.